src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2014.ProvideDynamicMessageContent_Invalid.sarif

```diff
@@ -17,7 +17,8 @@
                 "NoPlaceholders": {
                   "text": "This message does not contain dynamic content."
                 }
-              }
+              },
+              "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html"
             }
```

Comment:
we removed these to make the test cases smaller, maybe this got back due to merge. Please remove these (same for other 3 files- 2014, 2015)

Reply:
oh haha! wait a second! now we have a rule which makes this field mandatory!!! lol, now i see why u put it back.

---
In reply to: [447983343](https://github.com/microsoft/sarif-sdk/pull/1949#discussion_r447983343) [](ancestors = 447983343)

Reply:
the new rule developed (2012), will show that as an error, since we are using it as verbose. That's why i added again.